### DISCUSSION TITLE
API Key section missing

### DISCUSSION THREAD

**Discussion by alexdelprete** [dani-garcia/vaultwarden#2185] [Q&A] [answered]
Hi,

I just migrated to Vaultwarden from Bitwarden, bakced up data and restored it to Vaultwarden via Portwarden. Everything seems to be working fine.

Checking all the settings, I noticed I'm missing the API Key section in Settings->My Account.

Bitwarden:
![image](https://user-images.githubusercontent.com/7027842/147614346-5007e289-f6df-481d-8c65-1e0b27adbc8b.png)

Vaultwarden:
![image](https://user-images.githubusercontent.com/7027842/147614289-336f8215-4901-4765-bcd4-6b0eaea5348e.png)

**Comment by BlackDex** [comment, marked as answer]
API Key is not (yet) supported by Vaultwarden, and thus hidden/removed.

**Comment by alexdelprete** [comment]
Thanks for the answer. Is that why Portwarden doesn't work to export data from Vaultwarden? I'm trying with this command, but it doesn't work:  `portwarden --passphrase xxxxxxxxxxxxxxxxxxxxx --filename backup.portwarden encrypt`

Same command works with Bitwarden's vault.
